src/model/src/PT3D_FIRE_DEFN.F

```diff
@@ -225,7 +225,6 @@ SUBROUTINE GET_PT3D_FIRE_EMIS ( JDATE, JTIME )
             M = PTEM_FIRE_MAP( S )
             IF ( M .GT. 0 ) THEN
                N = PTEM_MAP( S )
-c           IF ( N .GT. 0 ) THEN
```

Comment:
@rmontuoro or @JianpingHuang-NOAA just confirming, you're removing an `IF..THEN` but I don't see an `END IF` getting removed as well.

Is this OK?

Reply:
ah i see my error. nevermind.